### ISSUE TITLE
Feedback.

### ISSUE THREAD

**Issue by Pengwin83** [the-abe-train/plurality#5]
First of all: I am a Dev myself and like the idea. I played/answered all available polls and I didnt know where else to place my feedback. All feedback given is in good faith that you will bring this game to more enjoyable level. You did something great here!

I found some issues that make this concept far less fun than it could be and some which are more like game-breaking:
* It feels like there can be multiple answeres with really low percentage and they highly depend on the users. So in theory you can end up with a poll having the top 20 answers with less then 60% which is really unsatisfying and makes it "unwinnable". Or is making the players guess the number 1 answer mandatory. 
  * Possible fix: Finding a secondary winning rule, if those conditions apply, e.g. Hitting 5-10 answers from the top20 or so.
  * Possible fix. Removing "less important" answers from the "good answer pool", e.g. "just 1 person answered this". 
    * To fix making poll takers feeling left out, when they see their answer was removed: Leave the answer in. Dont punish by reduce guesses. Make clear the answer was given, but only by so few people that the game didnt took it to the top100 answers, which is actually used for the game.
* Many answers seem to aim for the same answers, but each only have one answer. Like answering "candy" for the halloween poll would basically the same as "sweets", "sweet" or any other more specific kind of candy.
  * Possible fix: When a poll is answered suggest similar answers to the user as a list he can pick from. Like displaying candy first, since already 5 users answered this. Maybe there is some avaible category mechanism out there you can use, so your poll game does not have to "learn" this by itself.
  * I dont see an easy fix for this. Currently it feels you only can sanitize the answers manually, which well....sucks when being a programmer trying to build a self-fueling poll game :-)
* I generally dislike having to register with email for playing more than 1 poll. But I expect to miss something here. I dont see what you are achieving by doing that.
* Winning or losing to a poll should give some kind of transparency what the missing or remaining answers were. Players are left with the feeling of that gap after gameover. They cant improve and see what they missed. I personally dislike that "loose end".
* Some polls are kind of pointless and require special knowledge. If you dont have it the persons answering the poll will answer just "something", which will reduce the quality of the answer. Like "what is the worth of bitcoin in 10 years" (or similar topic. I dont remember).
  * Guessing exact numbers isnt fun here. You will just randomly hit answers until you reach 60%.
* Option to see more "numbers". Like total answer points in relation to the current answers. Maybe also the option to see the absolute number and percentage at the same time.

I hope this is helpful for you. (playing more globle now. I want cities, since i already know all countries. please make it happen. or let me know if I can support you there) :-)

**Comment by Pengwin83** [comment]
I found another poll: The #15. 
* The percentages appear to split about 60 percent in the top 4 answers. That means to get a real chance of hitting 60% you have to hit AT LEAST 3 right answers from the top4 (concluding you get the remaining 15 percent with good guesses from the remaining answers (hitting many more top20 answers). 
* I also noticed that "sweat" and "sweating" are 2 independent answers (answer 2 ~15% % answer 7 ~ 3%)
EDIT:
* After finishing I noticed that "sweaty" was also a valuable third variant I missed. And since I covered "rain" I missed to get "weather" as a valuable guess.
* In conclusion this poll is a perfect example were 2 answers (weather and sweat) cover 60% in total, but are split over 5 answers. And you kind of have to guess all of them to "win". This feels unfair from the player perspective.
* I also noticed your effort on the results page. Great job on the game over transparency. 👍

**Comment by simonpotatis** [comment]
On the result page, when you see the top survey responses, and you change from # to %, the Remaining doesnt show the remaining precentage of the top answers, it shows the remaining precentage of the answers i guessed.

**Comment by the-abe-train** [comment]
Thank you so much for the feedback! I meant to respond to your points sooner but my first draft got erased and I'm just picking it up again now.

One of the hardest problems in making this game is combining similar answers. I'm using a library called `wink-lemmatizer` to join together words on a grammatical basis (i.e. plurals and conjugations) but it's not good enough for a lot of use cases, including those you mentioned. I am currently looking into partnering with an NLP AI startup to tackle this problem. I have high hopes that will come up with an exciting and effective solution!

I encourage people to make accounts for a couple reasons, but most importantly, you need to have an account to submit a response so that I have a record of one response per player. If people aren't submitting responses, then the game becomes unplayable, thus I force you to have an account to do most things in the game.

Some of the Survey questions are very general, and some are a little more esoteric. This is practically unavoidable, given that there needs to be a new Survey every day, but I also don't think this is a bad thing. Even if people will skip the Surveys that they  have no connection to, I think that they will more greatly enjoy the Surveys that are more related to their interests. I hope that players will appreciate the variability as well; I don't want the game to get stale with super generic questions!

Lastly, I will acknowledge that you make a good point about some Surveys being "unwinnable". In some Surveys, it's very easy to get to 60%, and in others it's nearly impossible. That said, I don't want players to think that the Surveys that are difficult to win are pointless. Rather, I want to encourage people to try to achieve the highest score possible while using their all their guesses, and to compare their scores with their friends, regardless of whether or not they win. Thus, I am thinking about ways to de-emphasize passing the winning threshold and shift the focus onto maximizing score. The solution could be to lower the winning threshold, or making it variable by Survey (like the max number of guesses), or even removing it all together. Open to suggestions here.

Again, I want to thank you for the thorough feedback. In the name of improving my games I am always open to recommendations and constructive criticism. While I wouldn't say all the concerns raised here are resolved, I am going to close this issue because having all those points in a single thread makes it difficult to have a focused conversation. If anyone would like to revisit or hash out any of the points discussed in the original post or in my response, please open a new issue.

Thank you!
